examples/autoqasm/6_Customize_gate_calibrations.ipynb

```diff
@@ -0,0 +1,391 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "dda432a3",
+   "metadata": {},
+   "source": [
+    "# Customize gate calibrations\n",
+    "\n",
+    "The pulse implementation of gates is called \"gate calibrations\". Gaining access to gate calibrations empowers you to experiment with quantum gate designs and execute quantum programs with customized, improved gate calibrations. This notebook shows you how to access the native gate calibrations provided by quantum hardware providers, and shows you how to define custom gate calibrations for your quantum programs.\n",
+    "\n",
+    "We start with basic imports, and initialize the Rigetti Aspen-M-3 device which is the targeted device for this example notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "30fd0b59",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "import braket.experimental.autoqasm as aq\n",
+    "from braket.experimental.autoqasm import pulse\n",
+    "from braket.experimental.autoqasm.instructions import rx, rz, measure\n",
+    "\n",
+    "from braket.aws import AwsDevice\n",
+    "from braket.devices import Devices\n",
+    "from braket.circuits import Gate, QubitSet, FreeParameter\n",
+    "from braket.pulse import DragGaussianWaveform\n",
+    "\n",
+    "device = AwsDevice(Devices.Rigetti.AspenM3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f9fdfd5e",
+   "metadata": {},
+   "source": [
+    "## Gate calibrations from hardware providers"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5d08537b",
+   "metadata": {},
+   "source": [
+    "To customize a gate calibration, you often need to be intimately familiar with how quantum gates are implemented with pulses. It is often a good place to start by viewing the native gate calibrations supplied by hardware providers. To do so, we first retrieve the information from the `gate_calibrations` attribute of a device. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "f1e5a16f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "provider_calibrations = device.gate_calibrations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "959c5fe8",
+   "metadata": {},
+   "source": [
+    "Even for the same gate, different physical qubits and angles usually have a different pulse implementation. So, each gate calibration is associated with certain physical qubits and angles. In the code snippet below, we retrieve the gate calibration of the Rx gate for physical qubit `$0` and `angle=pi/2`."
```

Comment:
```suggestion
    "Even for the same gate, different physical qubits and angles usually have a different pulse implementation. Because of this, a gate calibration may be associated with specific physical qubits and angles. In the code snippet below, we retrieve the gate calibration of the Rx gate for physical qubit `$0` and `angle=pi/2`."
```